skunk.gemspec

```diff
@@ -48,8 +48,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "minitest-stub_any_instance", "~> 1.0.2"
   spec.add_development_dependency "minitest-stub-const", "~> 0.6"
   spec.add_development_dependency "rake", "~> 13.0"
-  spec.add_development_dependency "reek", "~> 6.1"
-  spec.add_development_dependency "rubocop", "~> 1.48"
```

Comment:
I would still list these 2 here (without a version restriction), so they show up as dev dependencies in rubygems, and then you keep the version definition in the specific Gemfiles